api-tests/src/steps/ipv-steps.ts

```diff
@@ -4,6 +4,7 @@ import { World } from "../types/world.js";
 import * as internalClient from "../clients/core-back-internal-client.js";
 import * as externalClient from "../clients/core-back-external-client.js";
 import * as criStubClient from "../clients/cri-stub-client.js";
+import { getAuditEvents } from "../clients/local-audit-client.js";
```

Comment:
Yeah fair enough - I sort of dislike `*` imports because they can unexpectedly pollute your namespace (same rationale as Java) and inhibit tree-shaking (hard to know which imports you actually need).

However - neither of those are really a concern here, and it might make it a bit clearer as you say.